internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/job.go

```diff
@@ -0,0 +1,72 @@
+package bridge
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keboola/go-client/pkg/keboola"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/plugin"
+	keboolasink "github.com/keboola/keboola-as-code/internal/pkg/service/stream/sink/type/tablesink/keboola"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/model"
+)
+
+func (b *Bridge) Job(k key.JobKey) op.WithResult[keboolasink.Job] {
+	return b.schema.Job().ForJob(k).GetOrErr(b.client)
+}
+
+func (b *Bridge) DeleteJob(k key.JobKey) *op.AtomicOp[keboolasink.Job] {
+	var active keboolasink.Job
+	return op.Atomic(b.client, &active).
+		// Entity must exist
+		Read(func(ctx context.Context) op.Op {
+			return b.schema.Job().ForJob(k).Get(b.client)
+		}).
+		// Delete
+		Write(func(ctx context.Context) op.Op {
+			return b.schema.Job().ForJob(k).Delete(b.client)
+		})
+}
+
+func (b *Bridge) createJob(ctx context.Context, token string, file plugin.File, storageJob *keboola.StorageJob) error {
+	modelJob := model.Job{
+		JobKey: key.JobKey{
+			SinkKey: file.SinkKey,
+			JobID:   key.JobID(storageJob.ID.String()),
+		},
+	}
+	b.logger.Debugf(ctx, "creating job: %v", modelJob)
+
+	keboolaJob := keboolasink.Job{
+		JobKey:        modelJob.JobKey,
+		StorageJobKey: storageJob.StorageJobKey,
+		Token:         token,
+	}
+	b.logger.Infof(ctx, "creating storage job: %v", keboolaJob)
```

Comment:
Shouldn't the job be an attribute rather than part of the message?

Reply:
I added it into `ctxattr`